src/styles/index.scss

```diff
@@ -1,3 +1,7 @@
-body {
+* {
   margin: 0;
 }
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document